Lib/openbakery/codetesting.py

```diff
@@ -201,22 +201,29 @@ def GLYPHSAPP_TEST_FILE(f):
     return glyphsLib.load(open(the_file, encoding="utf-8"))
 
 
-def assert_PASS(check_results, reason="with a good font...", ignore_error=None):
-    print(f"Test PASS {reason}")
```

Comment:
here you removed the print, which is meant to show up on the build log and facilitate debugging

Reply:
I removed them because they aren't necessary. When a pytest run fails there is plenty of other info to support the debugging. Those prints are just a distraction.